doc/Support/Device-Sensors.md

```diff
@@ -0,0 +1,49 @@
+source: Support/Device-Sensors.md
+
+## Device Sensors 
+
+LibreNMS has a standard for device sensors they are split into categories.
+This doc is to help users understand device sensors in general, if you need help with developing sensors for a device please see the Contributing + Developing section. 
+
+### Health Sensors
+
+The High and Low values of these sensors can be edited in Web UI by going to the device settings -> Health. There you can set your own custom
+High and Low values. List of these sensors can be found here [Link](../Developing/os/Health-Information.md)
+
+**Note** Some values are defined by the manufactures and other calculated when you add the device into librenms and some are set to null leaving it up
```

Comment:
None should be null, any in the code that are null will be auto calculated so you only have device defined or calculated ones.